errai-common/pom.xml

```diff
@@ -84,10 +85,6 @@
       <artifactId>gwt-slf4j</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-    </dependency>
 
```

Comment:
this dependency removal is not handled here in the PR correctly. There are multiple usages of commons-lang3 classes as transitive dependencies in errai-codegen or somewhere else too